pages/index.js

```diff
@@ -1,27 +1,74 @@
 import Form from "../components/Form";
-import { useContext } from "react";
-import { DataContext } from "./_app";
+import { useContext, useState, useEffect } from "react";
+import { DataContext, WatchlistContext } from "./_app";
 import Navigation from "../components/Navigation";
 import styled from "styled-components";
+import Movie from "../components/Movie";
+import getRandomIndexFromArray from "../utils/getRandomIndexFromArray";
+import { suggestionText } from "./api/suggestionText";
+import Link from "next/link";
+
+const StyledSectionHeader = styled.section`
+  padding-left: 30px;
+`;
+
+const StyledSectionForm = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+`;
+
+const StyledMoviePick = styled.section`
+  margin-top: 40px;
+`;
+
+const StyledNoMoviePick = styled.section`
+  padding-left: 30px;
+  width: 350px;
+`;
+
+const StyledHeader = styled.h2`
+  padding-left: 30px;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+`;
+
+const StyledLine = styled.hr`
+  margin-top: 20px;
+  width: 140px;
+  border-top: 1px solid lightgrey;
+`;
 
 export default function Home() {
   const { handleFormSubmit, movies } = useContext(DataContext);
+  const { watchlist } = useContext(WatchlistContext);
+  const [runtime, setRuntime] = useState(0);
 
-  const StyledSectionHeader = styled.section`
-    padding-left: 30px;
-  `;
-
-  const StyledSectionForm = styled.section`
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 60px;
-  `;
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(
+          `https://api.themoviedb.org/3/movie/${movie.id}?api_key=${process.env.NEXT_PUBLIC_API_KEY}&language=eng-US`
+        );
+        if (response.ok) {
+          const data = await response.json();
+          setRuntime(data.runtime);
+        } else {
+          throw new Error("Something went wrong");
+        }
+      } catch (error) {
+        console.log(`Error: ${error.message}`);
+      }
+    }
+    fetchData();
+  }, []);
 
-  const StyledLine = styled.hr`
-    width: 140px;
-    border-top: 1px solid lightgrey;
-  `;
+  const randomMovie = getRandomIndexFromArray(watchlist);
+  console.log(watchlist[getRandomIndexFromArray(watchlist)]);
```

Comment:
Do not forget to remove the console.log() later on 😉

Reply:
Thanks for the reminder! 😬